### ISSUE TITLE
[CI] Code linting

### ISSUE THREAD

**Issue by TheElan** [SinTh0r4s/ExampleMod1.7.10#20]
Add plugin(s) which perform static analysis and include them in CI step along with action which will pinpoint lines in question. (E.g. stream not closed, infinite loop, etc)

**Comment by bombcar** [comment]
Can a json validator be added to the quest json before merge? Is that a thing?

**Comment by TheElan** [comment]
> Can a json validator be added to the quest json before merge? Is that a thing?

yes, there are tools for that, e.g. [in python](https://stackoverflow.com/a/47289898) and even [gradle plugins](https://github.com/alenkacz/gradle-json-validator)
